### ISSUE TITLE
How to run this in development?

### ISSUE THREAD

**Issue by iamajvillalobos** [stellar/account-viewer#46]
Hi there, 

I'm new to Stellar and wanted to contribute, but I can't seem to get it working. npm version is `5.6`

```bash
account-viewer master % interstellar develop
events.js:136
      throw er; // Unhandled 'error' event
      ^

Error: spawn /usr/local/lib/node_modules/interstellar/bin/../../../node_modules/gulp/bin/gulp.js ENOENT
    at _errnoException (util.js:999:13)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:201:19)
    at onErrorNT (internal/child_process.js:389:16)
    at process._tickCallback (internal/process/next_tick.js:152:19)
    at Function.Module.runMain (module.js:703:11)
    at startup (bootstrap_node.js:194:16)
    at bootstrap_node.js:618:3
account-viewer master %
```

**Comment by klaaspieter** [comment]
I took a brief look and this appears to be a problem with interstellar. It's looking for [`node_modules/gulp`](https://github.com/stellar/interstellar/blob/d04a86576d809cf01d3c4977940ca75fe62a53ba/bin/interstellar.js#L3), but on my system gulp was installed at `node_modules/gulp-cli`.

Making that change I get a new error that I haven't been able to figure out yet:

```
interstellar develop
/usr/local/lib/node_modules/interstellar/node_modules/socket.io/lib/store.js:35
Store.prototype.__proto__ = EventEmitter.prototype;
                                         ^

TypeError: Cannot read property 'prototype' of undefined
    at Object.<anonymous> (/usr/local/lib/node_modules/interstellar/node_modules/socket.io/lib/store.js:35:42)
    at Module._compile (module.js:641:30)
    at Object.Module._extensions..js (module.js:652:10)
    at Module.load (module.js:560:32)
    at tryModuleLoad (module.js:503:12)
    at Function.Module._load (module.js:495:3)
    at Module.require (module.js:585:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/interstellar/node_modules/socket.io/lib/manager.js:16:13)
    at Module._compile (module.js:641:30)
```

**Comment by klaaspieter** [comment]
It looks like [`EventEmitter` is deprecated](https://superuser.com/a/1158316)

**Comment by nengine** [comment]
I am having similar, issue and not so sure what to do!

**Comment by sgehrman** [comment]
Same problem.

**Comment by bartekn** [comment]
I've just published [`interstellar@0.0.11`](https://www.npmjs.com/package/interstellar) which should fix the errors. Make sure you're using Node `v6.9.*`. You can use [`nvm`](https://github.com/creationix/nvm) to have multiple Node versions on your system.

**Comment by nengine** [comment]
Thank you @bartekn !
